src/index.ts

```diff
@@ -5,6 +5,8 @@ import { set, get, unset } from "lodash";
 export { IDriver } from "./drivers/IDriver";
 export { SqliteDriver } from "./drivers/SqliteDriver";
 export { MySQLDriver } from "./drivers/MySQLDriver";
+export * from "./drivers/MemoryDriver";
+export * from "./drivers/JSONDriver";
```

Comment:
You can change those to export the Specific class.
It allows more control over what we export from the package